### PR TITLE
Add api request libraries

### PR DESCRIPTION
**Pull request by radum**
I have added an example for the validation flow on how the routing, controller, and service for calling the API should work.

### CODE REVIEW

File: apps/web/.env.development [hunk 1]
```diff
@@ -23,3 +23,6 @@ HTTPS_ENABLED=false
 PORT=3000
 # Default : 3443
 HTTPS_PORT=3443
+
+# API Endpoint
+API_ENDPOINT=http://localhost:3000
```
Comment (mayya-bond-kc) [inline apps/web/.env.development:28]: `API_ENDPOINT` is not super clear, maybe rename to `API_HOST`?

**Review by mayya-bond-kc — APPROVED** [review]
LGTM

File: apps/web/.env.development (hunk 1 above)
Comment (radum) [reply apps/web/.env.development:28]: I will update.

**Pull request merged**